core/utils.py

```diff
@@ -35,9 +35,15 @@
 import urllib.request
 import zlib
 
-from core import feconf
 from core.constants import constants
 
+# The third party library path is defined in common.py,
```

Comment:
The changes to this file are temporary so that I could commit my changes, will remove them once #15608 gets merged.

Reply:
Removed this change as the above PR got merged.